listenbrainz_spark/config.py.sample

```diff
@@ -50,3 +50,7 @@ LOG_SENTRY = {
 # String + UUID
 # The following var defines the string of which the model id is made up of.
 MODEL_ID_PREFIX = 'listenbrainz-recommendation-model'
+
+FTP_SERVER_URI = 'ftp.musicbrainz.org'
+TEMP_LISTENS_DIR = 'listenbrainz-dump-88-20190930-000002-full/'
```

Comment:
These are strange example dir names... Why did you choose those?

Reply:
I have hardcoded the dump names here. These configs won't be used when we are clear about full and incremental dumps and hence the name.

Reply:
Ok, got it.